scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -12,10 +12,25 @@ import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
+import scala.meta.internal.semanticdb.{SymbolInformation, TypeRef, TypeSignature}
 import scalafix.internal.v0._
 
 class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig) extends Printer {
   def pprint(tpe: s.Type): Unit = {
+    def equivalent(name: Option[n.Name], sym: String): Boolean = {
+      name
+        .flatMap(env.lookupInfo)
```

Comment:
I suggest we keep the high-level contract of `Scope` and `Env` focused on name to symbol lookups, so I think it'd be better to revert the changes to `Scope` and `Env`. In my experience, this separation of concerns really helps (that's what we do in Rsc, too).

Where do we get infos then? In this particular context, which seems to be the only use for `lookupInfo`, symbol to info lookups can be performed via `index.symbol` which is a `SymbolTable`.